sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
+**Title:** A Decentralized Two-Way Bitcoin Peg  
+**Type:** Standard  
+**Status:** Draft  
+
+## Abstract
+sBTC is a novel digital asset that lets you move Bitcoin in and out of the Stacks blockchain. With sBTC, users can interact with Clarity smart contracts, which enable Bitcoin applications such as payments, decentralized lending, decentralized exchanges, and BTC-backed stablecoins.
+
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers. When BTC is locked on the Bitcoin L1, an equivalent amount of sBTC is issued on the Stacks layer, ensuring a consistent 1:1 ratio of sBTC:BTC. Users can redeem their sBTC at any time by submitting a withdrawal request. Once the request is processed by sBTC signers, BTC is returned to the user’s specified address on the Bitcoin L1.
+
+sBTC will have a crucial role in scaling Bitcoin, as well as introducing new and innovative functionalities to users, growing the size of the overall Bitcoin ecosystem. This SIP aims to describe the sBTC system, the process for signer selection, and the features available in the initial release compared to subsequent versions of sBTC. It does not attempt to describe the low-level technical details of any subsequent release, which will be provided in a future SIP.
+
+## Introduction
+
+## Glossary
+
+| Term                     | Definition |
+|--------------------------|------------|
+| SIP-10 Token             | A token on the Stacks blockchain that adheres to the fungible token standards outlined in SIP-10. |
+| sBTC                     | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
+| sBTC operation           | An operation that initiates some action from the sBTC protocol. |
+| Bitcoin holder           | A person or system holding BTC, interested in acquiring sBTC. |
+| sBTC holder              | A person or system holding sBTC, interested in turning it back into BTC. |
```

Comment:
Same here -- how is `sBTC holder` so ambiguous that it needs a definition?

Reply:
+1 for verbosity. agreed it doesn't add anything new to the accepted definition, but i also don't see any reason to remove outside of a goal of using less chars